pkg/sortition/sortition_test.go

```diff
@@ -0,0 +1,182 @@
+package sortition
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-log"
+	"github.com/keep-network/keep-core/pkg/internal/testutils"
+	"github.com/keep-network/keep-core/pkg/sortition/internal/local"
+)
+
+const (
+	testOperatorAddress        = "0x3c5eBAcFe5aE12D82d43602a12b8bBb76b893CfA"
+	testStakingProviderAddress = "0x80C63B577DC79B2432357BECC5b431dfb8E181DD"
+
+	statusCheckTick = 10 * time.Millisecond
+)
+
+func TestMain(m *testing.M) {
+	if os.Getenv("PRINT_LOGS_IN_TEST") == "true" {
+		err := log.SetLogLevel("*", "DEBUG")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "logger initialization failed: [%v]\n", err)
+			os.Exit(-1)
+		}
+	}
+
+	os.Exit(m.Run())
+}
```

Comment:
Can we drop a short comment explaining what happens here? Might be hard to grasp at the first sight.

Reply:
Good point, done in https://github.com/keep-network/keep-core/pull/3077/commits/05f10d92acbc9560cd398a08b1000b2fa2c6679e.